lib/config/dynamo_db.rb

```diff
@@ -0,0 +1,8 @@
+require 'dry-configurable'
+
+class DynamoDBConfig
+  extend Dry::Configurable
+
+  setting :app_name, default: 'app_name'
+  setting :table_name, default: 'logs'
```

Comment:
We intend for `app_name` to be available for all loggers, right? Should we move this to `EventTracer`?

Reply:
True. Not used for now but makes more sense for future use!